From d52ee79002b228d8af627d9e68b4257aa3d54bd8 Mon Sep 17 00:00:00 2001
Message-Id: <d52ee79002b228d8af627d9e68b4257aa3d54bd8.1635185056.git.stefan@agner.ch>
In-Reply-To: <7bc8a611131f89c7773e75863be58a682bfefa46.1635185056.git.stefan@agner.ch>
References: <7bc8a611131f89c7773e75863be58a682bfefa46.1635185056.git.stefan@agner.ch>
From: Stefan Agner <stefan@agner.ch>
Date: Mon, 19 Apr 2021 10:59:03 +0200
Subject: [PATCH 11/11] package/rpi-firmware: bump version to 1.20210527

Keep rpi-firmware up-to-date with the kernel version bump (tag
1.20210527).

Signed-off-by: Stefan Agner <stefan@agner.ch>
---
 package/rpi-firmware/rpi-firmware.hash | 2 +-
 package/rpi-firmware/rpi-firmware.mk   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/package/rpi-firmware/rpi-firmware.hash b/package/rpi-firmware/rpi-firmware.hash
index 0fd7a87bea..5130caf057 100644
--- a/package/rpi-firmware/rpi-firmware.hash
+++ b/package/rpi-firmware/rpi-firmware.hash
@@ -1,3 +1,3 @@
 # Locally computed
-sha256  d996fd1a2d53f692bf036d25981ecc690c126c88c51f60165fc7a568ed128b8f  rpi-firmware-fcf8d2f7639ad8d0330db9c8db9b71bd33eaaa28.tar.gz
+sha256  32fb5cc72703247622839c3cb4df2f369f91d2185770df867ac9758283bf5380  rpi-firmware-1eb8921ae7158249894f89e8bb6c4bfb04ec7377.tar.gz
 sha256  c7283ff51f863d93a275c66e3b4cb08021a5dd4d8c1e7acc47d872fbe52d3d6b  boot/LICENCE.broadcom
diff --git a/package/rpi-firmware/rpi-firmware.mk b/package/rpi-firmware/rpi-firmware.mk
index ea82836900..ac4c4a0788 100644
--- a/package/rpi-firmware/rpi-firmware.mk
+++ b/package/rpi-firmware/rpi-firmware.mk
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-RPI_FIRMWARE_VERSION = fcf8d2f7639ad8d0330db9c8db9b71bd33eaaa28
+RPI_FIRMWARE_VERSION = 1eb8921ae7158249894f89e8bb6c4bfb04ec7377
 RPI_FIRMWARE_SITE = $(call github,raspberrypi,firmware,$(RPI_FIRMWARE_VERSION))
 RPI_FIRMWARE_LICENSE = BSD-3-Clause
 RPI_FIRMWARE_LICENSE_FILES = boot/LICENCE.broadcom
-- 
2.33.1

